geometry_msgs/Point point_in_cam
float64 yaw
bool primmitive # 0 -> grasp; 1 -> suck
---
geometry_msgs/Pose result_pose
